cves/CVE-2018-7536.yml

```diff
@@ -15,24 +15,24 @@ curated_instructions: |
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: True
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that the
   CVE was created.  Leave blank if no date is given.
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: 2018-03-06
```

Comment:
Could you also investigate a little further here? Similar to my other comment, it seems like the vulnerability also was reported a little earlier if commits happened to fix this 2-24-2018